Automattic/wp-calypso
# Tailored Flows: Update Newsletter flow

#### Proposed Changes

Fix the newsletter flow following the latest changes in the designs

#### Testing Instructions



1. Pull branch and `yarn start`
2. Start at the beginning of the flow http://calypso.localhost:3000/setup?flow=newsletter
 * test with logged in and logged out
 * test purchase and free plan
3. Check the other /start flow to make sure there was no regression
